app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/protectiveword/ProtectiveWordFragment.kt

```diff
@@ -59,12 +82,25 @@ class ProtectiveWordFragment : Fragment(R.layout.fragment_protective_word) {
         observeMedicationRecords()
     }
 
+    private fun inputUpperCase() {
+        binding.etProtectiveWord.addTextChangedListener(textWatcher)
+    }
+
+    private fun setToUpperCase(editable: Editable) {
```

Comment:
Don't create a special method for the thing that is not going to be re-used.

Reply:
removed method.